### PR TITLE
Improve install documentation

### PR DESCRIPTION
**Pull request by mcollardanuy**
Changes to main readme:
* Uses of DeezyMatch
* Installation from source code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +23,11 @@
     <br/>
 </p>
 
-DeezyMatch can be applied for performing the following tasks:
+DeezyMatch can be applied as a component of the following tasks:
 
-- Fuzzy string matching
```
Comment (kasra-hosseini) [inline README.md:28]: @mcollardanuy What about Fuzzy String Matching?

File: README.md [hunk 2]
```diff
@@ -23,12 +23,11 @@
...
 - Record linkage
-- Candidate selection for entity linking systems
-- Toponym matching
```
Comment (kasra-hosseini) [inline README.md:31]: Should we keep this as we have used DM for toponym matching?

File: README.md [hunk 3]
```diff
@@ -23,12 +23,11 @@
     <br/>
 </p>
 
-DeezyMatch can be applied for performing the following tasks:
+DeezyMatch can be applied as a component of the following tasks:
```
Comment (kasra-hosseini) [inline README.md:26]: Minor: `DeezyMatch can be used in the following tasks:` What do you think?

**Review by kasra-hosseini — COMMENTED** [review]
@mcollardanuy Thanks for the PR. I left some (minor) comments.

File: README.md (hunk 3 above)
Comment (mcollardanuy) [reply README.md:26]: Hi @kasra-hosseini! Sorry, I should have explained why I suggested changing this paragraph:
* Regarding the main sentence ("DeezyMatch can be XXX the following tasks"): I've realized many people think DeezyMatch on its own can perform these tasks (and I think our previous phrasing was misleading it this regard), but this is not really the case: DM can make these tasks work better if fuzzy string matching is needed. So I wanted to make this clearer: that DM can be a helpful component in tasks requiring fuzzy string matching, such as X, Y and Z.
* Then, as for the list of tasks, the new list of tasks I suggested consists of downstream tasks only (entity linking, record linkage, query expansion through correction, and I'm sure there must be more). Fuzzy string matching is a common problem in these downstream tasks, which is where DeezyMatch can help. Toponym matching is just one type of fuzzy string matching, so I thought we didn't really need it.

What do you think? I add @fedenanni for advise as well.

Comment (kasra-hosseini) [reply README.md:26]: @mcollardanuy Thanks for the clarification. What about the following wording (or a version of that):

```
DeezyMatch can be used in the following tasks:

- Fuzzy string matching
- Candidate ranking/selection
- Query expansion
- Toponym matching

Or as a component in tasks requiring fuzzy string matching and candidate ranking, such as:
- Record linkage
- Entity linking 
```

**Comment by kasra-hosseini** [comment]
@mcollardanuy Thanks again for the changes. Re the first paragraph, I still think it would be good to be a bit more explicit about DM's features. My suggestion is something like:

```
DeezyMatch can be used in the following tasks:

- Fuzzy string matching
- Candidate ranking/selection
- Query expansion
- Toponym matching

Or as a component in tasks requiring fuzzy string matching and candidate ranking, such as:
- Record linkage
- Entity linking 
```

File: README.md (hunk 3 above)
Comment (mcollardanuy) [reply README.md:26]: Hi @kasra-hosseini, this looks good to me!

Comment (kasra-hosseini) [reply README.md]: Great, thanks @mcollardanuy . Changed the README here: https://github.com/Living-with-machines/DeezyMatch/pull/110/commits/9a91e29eef6d8d5e5d99979f0b91cf85f441e556

**Pull request merged**